### DISCUSSION TITLE
[Script request] Mealie

### DISCUSSION THREAD

**Discussion by CrazyWolf13** [community-scripts/ProxmoxVE#32] [Request script] [answered]
### Application Name

Mealie

### Website

https://mealie.io/

### Description

Mealie is an intuitive and easy to use recipe management app. It's designed to make your life easier by being the best recipes management experience on the web and providing you with an easy to use interface to manage your growing collection of recipes. 

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.

**Comment by johndoe0815** [comment]
Also wondering that Mealie is still not there, but Tandoor is. However Tandoor I was also able to set up via Docker/Portainer LXC with even lower system resources, while I am not able to set up Mealie, always getting strange errors in Portainer.

**Comment by MickLesk** [comment]
## Statement:

Mealie currently only runs with a deprecated Node 16 version, I have written with the developers and they have massive problems pulling nuxt from version 2 to 3. They have tried several times but each time it breaks the frontend.

So announcement to the community: 
As long as it is not a current NODE version, we will not implement mealie for security reasons.

If there are any capable Nuxt developers here, please contact Mealie in the Discord, maybe you can help 👍

**Reply by oOStroudyOo** [reply]
I assume this is the pull request we're waiting for before we consider adding Mealie to the community scripts?

https://github.com/mealie-recipes/mealie/pull/5184

**Reply by MickLesk** [reply]
Waiting for merge, i Test after this with my local deployment

**Comment by qanadin** [comment]
Has anybody looked into this recently?

**Reply by CrazyWolf13** [reply]
@qanadin 
Like writte just a post above your reply, we re waiting for this PR to be merged tested and released, as soon as this is the case, we can look into rebuilding the script:
https://github.com/mealie-recipes/mealie/pull/5184

**Reply by Oettinger77** [reply]
Ist there an ETA for that? Don't wanna push, but something like next month**s**, sometime 2025 or even 2026.
Cause i want to start to use Mealie with this as a fresh start (aka i don't wanna use Docker).


Regards

**Reply by MickLesk** [reply]
if the devs of mealie ready. maybe days, weeks, months

**Reply by raenye** [reply]
@MickLesk
https://github.com/mealie-recipes/mealie/pull/5184 was merged yesterday.

**Reply by MickLesk** [reply]
Yes, we know

**Reply by MickLesk** [reply, marked as answer]
Script is done (atm as git clone - because missing release file of the new release).

can be tested - please give feedback in this Thread: (there is the bash call) 
https://github.com/community-scripts/ProxmoxVED/issues/607


Reminder: Thats only an test, if the release is available the script will be refactored to crawl the release package - **no productive usage! Only TEST**

Remark: if non verbose not work - try verbose. its an "dirty" script for testing

**Comment by MickLesk** [comment]
Merged into Main, Have fun!
